### PR TITLE
fix s p o o k y brightness knob action 👻

### LINKED ISSUE #722: [BUG] brightness knobs scroll themselves down with no user input
STARTED ONCE THE LATEST PATCH CAME OUT
**Mod Version**
 2.0 stable release 

**Describe the bug**
 cockpit brightness knobs turn themselves down with no input during flight. For example on a normal flight I was just in ALT CRZ chilling and not touching any keys or changing any views, hands off the keyboard, and brightness knobs for bot PFD and NDs and the FCU sporadically scroll themselves down, making the flying experience miserable.￼

**To Reproduce**

Conduct normal flight. I flew KPAE to KSEA 

1. Just fly a normal flight. Occasionally use mouse wheel to change zoom or spin knobs
2.
3.

**Expected behavior**

Shouldn't happen

**Actual behavior**
 happened

**References**

**Additional context**

Discord username (if different from GitHub):

### PR DESCRIPTION
**Pull request by AdenFlorian**
Fixes #722

**Summary of Changes**

So, it seems that when other planes load in on your machine, their init values for light knobs get run on your plane.

It also seems the traffic models for the planes have their own init values.

My temporary solution (until asobo fixes this in native code) is to change the potentiometer IDs of all the affected knobs. I added them by 70 to get them as high as possible so as not to conflict with knobs in other planes.

We did some pretty good testing before and after wth other players in A320's using my fix and without, and this fix seems solid.

**Screenshots (if necessary)**

n/a

**Additional context**

### CODE REVIEW

**Comment by Venorcis** [comment]
Nice! It seems the same bug affects way more than just the screen brightness knobs though sadly (e.g. #732). https://np.reddit.com/r/MicrosoftFlightSim/comments/iw6mis/plane_exorcisms_or_how_i_seem_to_have_reduced_or/ also mentions AP failures happening because of leaking values from (ground) AI traffic.

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js [hunk 1]
```diff
@@ -87,17 +87,17 @@ class A320_Neo_EICAS extends Airliners.BaseEICAS {
...
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:84","FLOAT64",0);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:85","FLOAT64",0);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:86","FLOAT64",0);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:87","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:88","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:89","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:90","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:91","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:92","FLOAT64",0.1);
+        SimVar.SetSimVarValue("LIGHT POTENTIOMETER:93","FLOAT64",0.1);
```
Comment (UncleClapton) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js:100]: Do we really need these still?

File: A32NX/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.js [hunk 2]
```diff
@@ -47,6 +47,12 @@ class NavSystem extends BaseInstrument {
             this.currFlightPlanManager.registerListener();
         }
         this.currFlightPlan = new FlightPlan(this);
+        if (typeof g_modDebugMgr != "undefined") {
+            g_modDebugMgr.AddConsole(null);
+        }
+        Include.addScript("/JS/debug.js", function () {
+            g_modDebugMgr.AddConsole(null);
+        });
```
Comment (UncleClapton) [inline A32NX/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.js:55]: If we're gonna add this, we should probably have it do nothing if debug isn't installed.
```suggestion
        if (typeof g_modDebugMgr !== "undefined") {
            Include.addScript("/JS/debug.js", function () {
                g_modDebugMgr.AddConsole(null);
            });
        }
```

**Review by UncleClapton — CHANGES_REQUESTED** [review]

File: A32NX/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.js (hunk 2 above)
Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/NavSystems/A320_Neo/A32NX_NavSystem.js:55]: oops, didn't mean to commit. but as you said might as well?

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js (hunk 1 above)
Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/A320_Neo_EICAS.js:100]: not sure, but i dont think that needs to be dealt with in this pr

**Comment by camilo625** [comment]
Seems to work fine

**Comment by AdenFlorian** [comment]
pushed a fix for dome light

**Review by ImenesFBW — APPROVED** [review]
Fixes the issue. You are my hero!

**Review by UncleClapton — APPROVED** [review]

**Comment by camilo625** [comment]
Works great now

**Pull request merged**